data/data-pipeline/data_pipeline/ipython/hud_eda_se_12_09_2021-revised-denominator.ipynb

```diff
@@ -0,0 +1,686 @@
+    "\n",
+    "\n",
+    "housing[\"denominator_pre\"] = (\n",
+    "    housing[OWNER_OCCUPIED_POPULATION_FIELD]\n",
+    "    + housing[RENTER_OCCUPIED_POPULATION_FIELD]\n",
+    "    - housing[OWNER_OCCUPIED_NOT_COMPUTED_FIELDS].sum(axis=1)\n",
+    "    - housing[RENTER_OCCUPIED_NOT_COMPUTED_FIELDS].sum(axis=1)\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Implementation by Saran <a id='saran'></a>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "OWNER_REVISED_DENOMINATOR_FIELDS =   [\n",
+    "            # Column Name\n",
+    "            #   Line_Type\n",
+    "            #   Tenure\n",
+    "            #   Household income\n",
+    "            #   Cost burden\n",
+    "            #   Facilities\n",
+    "            \"T8_est3\",\n",
+    "            #   Subtotal\n",
+    "            #   Renter occupied\tless than or equal to 30% of HAMFI\n",
+    "            #   All\n",
+    "            \"T8_est16\",\n",
+    "            #   Subtotal\n",
+    "            #   Renter occupied\tgreater than 30% but less than or equal to 50% of HAMFI\n",
+    "            #   All\n",
+    "            \"T8_est29\",\n",
+    "            #   Subtotal\n",
+    "            #   Renter occupied\n",
+    "            #   greater than 50% but less than or equal to 80% of HAMFI\n",
+    "            #   All\n",
+    "        ]\n",
+    "\n",
+    "RENTAL_REVISED_DENOMINATOR_FIELDS =   [\n",
+    "            # Column Name\n",
+    "            #   Line_Type\n",
+    "            #   Tenure\n",
+    "            #   Household income\n",
+    "            #   Cost burden\n",
+    "            #   Facilities\n",
+    "            \"T8_est69\",\n",
+    "            #   Subtotal\n",
+    "            #   Renter occupied\tless than or equal to 30% of HAMFI\n",
+    "            #   All\n",
+    "            \"T8_est82\",\n",
+    "            #   Subtotal\n",
+    "            #   Renter occupied\tgreater than 30% but less than or equal to 50% of HAMFI\n",
+    "            #   All\n",
+    "            \"T8_est95\",\n",
+    "            #   Subtotal\n",
+    "            #   Renter occupied\n",
+    "            #   greater than 50% but less than or equal to 80% of HAMFI\n",
+    "            #   All\n",
+    "        ]\n",
+    "\n",
+    "# New computation:\n",
+    "# (\n",
+    "#     # of Owner Occupied Units Meeting Criteria\n",
+    "#     + # of Renter Occupied Units Meeting Criteria\n",
+    "# )\n",
+    "# divided by\n",
+    "# (\n",
+    "#     Total # of Owner Occupied Units that meet criterion for poverty\n",
+    "#     + Total # of Renter Occupied Units that meet criterion for poverty\n",
+    "#     - # of Owner Occupied Units with HAMFI Not Computed\n",
+    "#     - # of Renter Occupied Units with HAMFI Not Computed\n",
+    "# )\n",
+    "\n",
+    "housing['denominator_post'] = housing[\n",
```

Comment:
It looks like something in this math is off? The ratio should be a number between 0 and 1 but yours goes up to 1.13 

<img width="395" alt="Screen Shot 2021-12-10 at 5 49 18 PM" src="https://user-images.githubusercontent.com/25443529/145651032-0089b5d6-1ae1-416a-80d1-2a61c92e03ab.png">

Reply:
@lucasmbrown-usds Thanks for pointing out that the ratios are greater than 1 for 20 census tracts. Some context:

![Screen Shot 2021-12-11 at 5 38 42 AM](https://user-images.githubusercontent.com/94847739/145673668-cffe5e6b-111b-4b16-8635-1898f893f5b6.png)
![Screen Shot 2021-12-11 at 5 43 57 AM](https://user-images.githubusercontent.com/94847739/145673713-be94c26b-ec4f-4131-aa29-bb909ccdd42f.png)

I can double-check the legend in CHAS. After that, I did want to run this by our contact in HUD.

Reply:
@lucasmbrown-usds - I was reviewing this issue #283. I would like to discuss this with you at some point.